cdap-ui/app/directives/dag-plus/my-dag.less

```diff
@@ -60,6 +60,7 @@
   &.selected {
     border-color: @node-color;
     color: white;
+    z-index: 2000;
```

Comment:
this z-index will be higher than the system is down notification?

Reply:
It shouldn't absolutely positioned elements with same z-index gets priority on the most recently added element.
<img width="1440" alt="Screen Shot 2021-02-16 at 10 29 00 PM" src="https://user-images.githubusercontent.com/1452845/108164846-6ff2d780-70a6-11eb-9f78-1317a2f55375.png">